### PR TITLE
Two small changes around the edges

### PR DESCRIPTION
**Pull request by pedrocamargo**
Adds the obvious field vot to the modes table and creates and adds a script for compiling that may be useful during development

### CODE REVIEW

**Comment by pedrocamargo** [comment]
@jamiecook , don't worry about the test failures. Those are known issues elsewhere in the development branch

File: tests/compile.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from os.path import join, dirname
+import os
+import subprocess
+
+pth = join(dirname(dirname(__file__)), 'aequilibrae', 'paths')
+
+os.chdir(pth)
+subprocess.Popen("python setup_Assignment.py build_ext --inplace", shell=True, stdout=subprocess.PIPE).stdout.read()
```
Comment (jamiecook) [inline tests/compile.py:8]: Is it worth using a substitution here

```suggestion
subprocess.Popen(f"{sys.executable} setup_Assignment.py build_ext --inplace", shell=True, stdout=subprocess.PIPE).stdout.read()
```

to make sure the right python is called.

File: aequilibrae/project/database_specification/tables/modes.sql [hunk 2]
```diff
@@ -1,7 +1,8 @@
 CREATE TABLE if not exists modes (mode_name   VARCHAR UNIQUE NOT NULL,
                                   mode_id     VARCHAR UNIQUE NOT NULL       PRIMARY KEY,
                                   description VARCHAR,
-                                  pce         NUMERIC        NOT NULL DEFAULT 0,
+                                  pce         NUMERIC        NOT NULL DEFAULT 1.0,
```
Comment (jamiecook) [inline aequilibrae/project/database_specification/tables/modes.sql:4]: This default changed from 0 -> 1 ... intentional?

Comment (jamiecook) [reply aequilibrae/project/database_specification/tables/modes.sql:4]: Guessing it probably is

**Review by jamiecook — APPROVED** [review]

**Pull request merged**